lang/java/reef-common/src/main/java/org/apache/reef/io/Tuple.java

```diff
@@ -41,13 +41,21 @@ public V getValue() {
 
   @Override
   public boolean equals(final Object o) {
```

Comment:
The sad part is that this is auto-generated code in most cases. Hopefully we can convince IntelliJ to generate code that complies with CheckStyle in the future.